Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PostgresEamDbSettings.java

```diff
@@ -370,6 +370,8 @@ public boolean initializeDatabaseSchema() {
         createReferenceSetsTable.append("org_id integer NOT NULL,");
         createReferenceSetsTable.append("set_name text NOT NULL,");
         createReferenceSetsTable.append("version text NOT NULL,");
+        createReferenceSetsTable.append("known_status integer NOT NULL,");
+        createReferenceSetsTable.append("read_only boolean NOT NULL,");
```

Comment:
Since we have publicly released the CR, I think we will need to add some upgrade code for this schema change. We can use org.sleuthkit.datamodel.CaseDbSchemaVersionNumber (CaseDbSchemaVersionNumber => DbSchemaVersionNumber). I suggest that we make a separate story for the upgrade code.

Reply:
JIRA-3245